src/main/webapp/app/overview/course-registration/course-registration-button/course-registration-button.component.ts

```diff
@@ -17,11 +18,14 @@ export class CourseRegistrationButtonComponent implements OnInit {
     userIsAllowedToRegister = false;
     loading = false;
 
+    confirmationTitle: string;
```

Comment:
Am I missing something? You only add this variable and initialize it, but I cannot see where you use its value